docs/deploy/android.rst

```diff
@@ -85,43 +85,45 @@ To deploy models on Android with reasonable performance, one has to cross-compil
   git clone https://huggingface.co/meta-llama/$MODEL_NAME \
             ./dist/models/
 
-**Compile Android-capable models**. Install TVM Unity compiler as a Python package, and then run the command below:
-
+**Compile Android-capable models**. Install TVM Unity compiler as a Python package, and then compile the model for android using the following commands:
 .. code-block:: bash
 
-  # Show help message
-  python3 -m mlc_llm.build --help
-  # Compile a PyTorch model
-  python3 -m mlc_llm.build \
-          --target android \
-          --max-seq-len 768 \
-          --model ./dist/models/$MODEL_NAME \
-          --quantization $QUANTIZATION
+  # convert weights
+  mlc_chat convert_weight ./dist/models/$MODEL_NAME/ --quantization $QUANTIZATION -o dist/$MODEL_NAME-$QUANTIZATION-MLC/
+  # create mlc-chat-config.json
+  mlc_chat gen_config ./dist/models/$MODEL_NAME/ --quantization $QUANTIZATION \
+    --conv-template llama-2 --context-window-size 768 -o dist/${MODEL_NAME}-${QUANTIZATION}-MLC/
+  # 2. compile: compile model library with specification in mlc-chat-config.json
+  mlc_chat compile ./dist/${MODEL_NAME}-${QUANTIZATION}-MLC/mlc-chat-config.json \
+      --device android -o ./dist/${MODEL_NAME}-${QUANTIZATION}-MLC/${MODEL_NAME}-${QUANTIZATION}-MLC-android.tar
```

Comment:
Let's perhaps remove `-MLC` in model libs (keep in weights) to make it consistent across platforms (lines 105 and 126 as well); makes the names shorter too